src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/collectionsgrid/view/CollectionsGrid.kt

```diff
@@ -24,43 +28,64 @@ class CollectionsGrid : Fragment() {
         importStylesheet<DefaultStyles>()
     }
 
-    override val root = vbox {
-        addClass(AppStyles.appBackground)
-        hgrow = Priority.ALWAYS
-        vgrow = Priority.ALWAYS
-        progressindicator {
-            visibleProperty().bind(viewModel.loadingProperty)
-            managedProperty().bind(visibleProperty())
-            addClass(CollectionGridStyles.contentLoadingProgress)
-        }
-
-        datagrid(viewModel.children) {
-            vgrow = Priority.ALWAYS
-            hgrow = Priority.ALWAYS
-            isFillWidth = true
-            addClass(AppStyles.appBackground)
-            addClass(CollectionGridStyles.collectionsContainer)
-            cellCache { item ->
-                card {
-                    addClass(DefaultStyles.defaultCard)
-                    cardfront {
-                        innercard(AppStyles.chapterGraphic()) {
-                            title = item.labelKey.toUpperCase()
-                            bodyText = item.titleKey
-                        }
-                        cardbutton {
-                            addClass(DefaultStyles.defaultCardButton)
-                            text = messages["openProject"]
-                            graphic = MaterialIconView(MaterialIcon.ARROW_FORWARD, "25px")
-                                    .apply { fill = AppTheme.colors.appRed }
-                            isDisableVisualFocus = true
-                            onMousePressed = EventHandler {
-                                viewModel.selectCollection(item)
+    private fun EventTarget.tabContents() : VBox {
```

Comment:
This is not in its final form. The tab contents will differ for scripture and notes.